lib/Service/PermissionsService.php

```diff
@@ -480,7 +481,7 @@ private function basisCheckById(int $elementId, string $nodeType, ?string &$user
 		return false;
 	}
 
-	private function userIsElementOwner(string $userId, $element): bool {
+	private function userIsElementOwner($element, string $userId = null): bool {
```

Comment:
Would be nice to type hint, maybe by introducing a base class or interface for all element classes